CMakeLists.txt

```diff
@@ -345,6 +344,7 @@ endif()
 #Add the various sub-projects
 if(${WITH_ABC})
     add_subdirectory(abc)
+    add_subdirectory(ace2)
```

Comment:
So moving ace2 above lines 352/352 does change which flags are used for these libraries.  Is that the intended behavior?

Reply:
Not at all. Thanks for the heads up

Reply:
@litghost should be resolved